grails-app/views/category/tree.gsp

```diff
@@ -11,16 +11,33 @@
             <g:if test="${flash.message}">
             	<div class="message">${flash.message}</div>
             </g:if>
-			<g:hasErrors bean="${categoryInstance}">
-				<div class="errors"><g:renderErrors bean="${categoryInstance}" as="list" /></div>
+            <g:if test="${flash.error}">
+                <div class="errors">${flash.error}</div>
+            </g:if>
+			<g:hasErrors>
```

Comment:
the indent feels weird here, doesn't it?

Reply:
It does, but I don't know what's wrong :man_shrugging: 
I used tabs in both cases and it looks good in the edditor